app/controllers/multipage_contents_controller.rb

```diff
@@ -0,0 +1,31 @@
+class MultipageContentsController < ApplicationController
+  def show
+    content_item_response = ContentStore.service.content_item(base_path)
+    @content = model_class.new(content_item_response.to_hash) if content_item_response
+
+    if @content.present?
+      if params[:part]
+        @current_part = current_part
+        redirect_to base_path unless @current_part
+      end
+    else
+      render_404
+    end
+  end
```

Comment:
If the user requests a part, but that part doesn't exist on the content item, would this 404?